### PR TITLE
Add iCade controller support for FR and DE on tvOS

### PR DESCRIPTION
**Pull request by viteinfinite**

### CODE REVIEW

**Comment by viteinfinite** [comment]
This also provides a proper fix for #601 when playing on an Apple TV.
Tested in French with my 8bitdo FC30 zero.

**Comment by viteinfinite** [comment]
🎉

File: Provenance/Controller/iCade/iCadeReaderView.m [hunk 1]
```diff
@@ -115,13 +126,32 @@ - (void)deleteBackward {
 
 #pragma mark - keys
 
+- (void)applyKeyMappingForCurrentLocale {
+#if TARGET_OS_TV
+    NSString *localeIdentifier = [[NSLocale currentLocale] localeIdentifier];
```
Comment (jasarien) [inline Provenance/Controller/iCade/iCadeReaderView.m]: Is there any reason why this is only done on Apple TV and not iOS?

**Comment by Canadadry** [comment]
@viteinfinite sorry, this is not working for other plateforme than AppleTv. Why did you needed to put de the `#if`statement ? 

If Have removed it and the code is still not working on my iPhone. My local identifier is `en_FR`

**Pull request merged**